packages/web-components/src/menu-item/menu-item.styles.ts

```diff
@@ -251,24 +251,25 @@ export const menuItemStyles: (context: ElementDefinitionContext, definition: Men
           border-color: ${SystemColors.HighlightText};
         }
         :host(:hover) .expanded-toggle,
-            :host(:hover) .checkbox,
-            :host(:hover) .radio,
-            :host(:${focusVisible}) .expanded-toggle,
-            :host(:${focusVisible}) .checkbox,
-            :host(:${focusVisible}) .radio,
-            :host([checked]:hover) .checkbox,
-            :host([checked]:hover) .radio,
-            :host([checked]:${focusVisible}) .checkbox,
-            :host([checked]:${focusVisible}) .radio {
+        :host(:hover) .checkbox,
+        :host(:hover) .radio,
+        :host(:${focusVisible}) .expanded-toggle,
+        :host(:${focusVisible}) .checkbox,
+        :host(:${focusVisible}) .radio,
+        :host([checked]:hover) .checkbox,
+        :host([checked]:hover) .radio,
+        :host([checked]:${focusVisible}) .checkbox,
+        :host([checked]:${focusVisible}) .radio {
           border-color: ${SystemColors.HighlightText};
```

Comment:
Should this be using outline?

Reply:
@chrisdholt

Reply:
I will actually remove this and other selectors that are styling this to look like a checkbox. 
It should not look like a checkbox. It should just be an indicator that the menu item is checked. 
Here is the screenshot of the change.
![image](https://user-images.githubusercontent.com/37851220/191623444-7ae90f8d-2b7c-4ec4-8439-6722414b526f.png)